website/addons/citations/provider.py

```diff
@@ -17,6 +17,17 @@ def __init__(self, provider_name):
     def serializer(self):
         pass
 
+    def check_credentials(self, node_addon):
+        valid = True
+        try:
+            node_addon.api.client
```

Comment:
I think I commented on this in the other PR, but this appears incorrect. Why would this raise an `HTTPError`?